README.md

```diff
@@ -87,14 +87,21 @@ from neural_compressor.torch.quantization import (
     prepare,
     convert,
 )
+
```

Comment:
Could you please help update the docker image:
`docker run -it --runtime=habana -e HABANA_VISIBLE_DEVICES=all -e OMPI_MCA_btl_vader_single_copy_mechanism=none --cap-add=sys_nice --net=host --ipc=host vault.habana.ai/gaudi-docker/1.18.0/ubuntu22.04/habanalabs/pytorch-installer-2.4.0:latest`